### PR TITLE
LocaleProvider add zh_HK, zh_MO support，组件支持繁体

### PR DESCRIPTION
**Pull request by xingyesh**
fixed issue #2849
support zh_HK, zh_MO

  Make sure that you follow antd's code convention.
  Run npm run lint and fix those errors before submitting in order to keep consistent code style.
  Rebase before creating a PR to keep commit history clear.
  Add some descriptions and refer relative issues for you PR.
Extra checklist:

if isBugFix :

  Make sure that you add at least one unit test for the bug which you had fixed.
elif isNewFeature :

  Update API docs for the component.
  Update/Add demo to demonstrate new feature.
  Update TypeScript definition for the component.
  Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/2857)

### CODE REVIEW

File: components/locale-provider/demo/basic.md [hunk 1]
```diff
@@ -104,6 +106,16 @@ class App extends React.Component {
...
       },
+      {
+        value: '中国-香港',
+        label: '中国香港繁体',
+        language: zh_HK,
+      },
+      {
+        value: '中国-澳门',
+        label: '中国澳门繁体',
+        language: zh_HK,
```
Comment (yutingzhao1991) [inline components/locale-provider/demo/basic.md]: zh_MO

Comment (xingyesh) [reply components/locale-provider/demo/basic.md]: 额。。。 最后一次commit没提交上去。。。

File: components/date-picker-view/locale/zh_HK.tsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* !TODO 暂时先用zh_CN 等rmc-data-picker local 增加zh_HK 再更新  */
```
Comment (doxiaodong) [inline components/date-picker-view/locale/zh_HK.tsx]: https://github.com/react-component/m-date-picker/blob/master/src/locale/zh_CN.tsx
用这个没问题么

**Comment by awmleer** [comment]
v2 已经不再维护了，这个 PR 先关闭了哈，如果有兴趣的话可以关注一下最新的 v5 版本～

**Pull request closed without merging**